package.json

```diff
@@ -18,15 +18,15 @@
   "homepage": "https://github.com/department-of-veterans-affairs/va-mobile-library#readme",
   "packageManager": "yarn@4.0.2",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^7.0.1",
-    "@typescript-eslint/parser": "^7.0.1",
-    "eslint": "^8.56.0",
+    "@typescript-eslint/eslint-plugin": "^7.17.0",
+    "@typescript-eslint/parser": "^7.17.0",
+    "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-sort-imports-es6-autofix": "^0.6.0",
     "eslint-plugin-testing-library": "^6.2.2",
-    "eslint-plugin-tsdoc": "^0.2.17",
-    "prettier": "3.2.5",
-    "typescript": "^5.3.3"
+    "eslint-plugin-tsdoc": "^0.3.0",
+    "prettier": "3.3.3",
+    "typescript": "~5.3.3"
```

Comment:
Why'd this go `^` to `~`?

Reply:
This is to satisfy the following expo warning:
```
The following packages should be updated for best compatibility with the installed expo version:
  typescript@5.5.4 - expected version: ~5.3.3
```

Leaving it as a carat will install `v5.5.4` which causes the warning to reappear.